M2/Macaulay2/packages/Macaulay2Doc/changes.m2

```diff
@@ -66,6 +66,10 @@ document {
 	       },
 	 LI { "functionality changed:",
 	      UL {
+		   LI {
+		       "$\\LaTeX$ typesetting for the web documentation is now rendered using ",
+		       HREF{"https://katex.org/","$\\KaTeX$"}, ", a JavaScript math typesetting library for browsers."
```

Comment:
Shouldn't this go in the section called "functionality improved:" and include information about how the appearance is better and more tex macros and features are supported?  For example, subscripts of subscripts presumably are now okay.

Reply:
I think this is fine. Feel free to move it.

Reply:
It's an important improvement, so its advantages need to be described and bragged about.  You're the one who can do that, not I.

Reply:
I prefer brief messages here rather than long ones, because longer messages are probably ignored, and I'm sure users will discover the improvements on their own very quickly.

Do you have a suggestion for what else the description should say? I'm trying to stay away from coding for a little bit, so please feel free to move it or add to it as you see fit.

Reply:
I wouldn't depend on users figuring that all out for themselves -- get them eager to try it.  And another destination for entries in our change log is our annual progress reports to the NSF, so it's worthwhile saying something that sounds positive rather than neutral.  What about something this, with the false bits corrected and more true bits added?:

$\\LaTeX$ typesetting for the web documentation is now rendered using ", HREF{"https://katex.org/","$\\KaTeX$"}, ", a JavaScript math typesetting library for browsers.  As a result, the subset of TeX that is correctly rendered in the HTML version of the documentation is substantially enlarged.  Matrices are now rendered as TeX would, rather than as stodgy old HTML tables.  Subscripts can now have subscripts, and fractions with horizontal bars work.  Many more characters in many more fonts are available.